x/subspaces/keeper/groups.go

```diff
@@ -106,6 +113,16 @@ func (k Keeper) DeleteUserGroup(ctx sdk.Context, subspaceID uint64, groupID uint
 
 // AddUserToGroup adds the given user to the group having the provided id inside the specified subspace.
 func (k Keeper) AddUserToGroup(ctx sdk.Context, subspaceID uint64, groupID uint32, user string) {
+	// Create account if user does not exist.
+	userAcc, err := sdk.AccAddressFromBech32(user)
+	if err != nil {
+		panic(err)
+	}
+	if !k.ak.HasAccount(ctx, userAcc) {
+		defer telemetry.IncrCounter(1, "new", "account")
+		k.ak.SetAccount(ctx, k.ak.NewAccountWithAddress(ctx, userAcc))
+	}
+
```

Comment:
Users in group now are required to have an account, it is useful when users have a group grant.